openmetadata-ui/src/main/resources/ui/webpack.config.prod.js

```diff
@@ -31,10 +31,6 @@ module.exports = {
   // Input configuration
   entry: ['@babel/polyfill', path.join(__dirname, 'src/index.tsx')],
 
-  cache: {
```

Comment:
@karanh37 , I believe we have this config in the dev config as well, do we need that?

Reply:
No this is not needed. Will remove that in another PR. 
Thanks for highlighting